src/components/modals/FeedbackModal.jsx

```diff
@@ -37,71 +41,92 @@ export const FeedbackModal = ({ visible, setVisible }) => {
     if (!visible) formik.current?.resetForm()
   }, [visible])
 
-  return (
-    <>
+  if (isFeedbackEditModalVisible) {
+    return (
       <FeedbackEditModal
         visible={isFeedbackEditModalVisible}
         setVisible={setFeedbackEditModalVisible}
         review={editedReview}
       />
-      {!isFeedbackEditModalVisible && (
-        <Modal
-          visible={visible}
-          onClose={() => setVisible(false)}
-          onEnterPress={() => setVisible(false)}
+    )
+  } else if (isDeleteModalOpen) {
```

Comment:
no need else if you are using return, it doesn't have any sense

if (true) return <><>

if (anotherTrue) return <><>

return <defaultJSX>